packages/wikitext/wikitext.1.0.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Julien SAGOT <julien.sagot@geneanet.org>"
+authors: "Maxence ABELA <maxence.abela+wikitext@gmail.com>"
+homepage: "https://github.com/geneanet/ocaml-wikitext"
+bug-reports: "https://github.com/geneanet/ocaml-wikitext/issues"
+dev-repo: "git+https://github.com/geneanet/ocaml-wikitext.git"
+synopsis: "Wikitext parser"
+build: [
```

Comment:
Since you are using `dune`, it is better to use the release calling convention here. I am going to make the change, please upstream also this